Improve text of second module (#11)
actions-workshop/actions-workshop: docs/002-basics-of-ci-with-actions.md

## 1 - Inspect the repository

> This repository contains a [React](https://reactjs.org/)-based application built with [Vite](https://vitejs.dev/). We aim to automate its testing and building in this segment of the workshop.

Reviewer: I don't like `segment
```suggestion
This repository contains a [React](https://reactjs.org/)-based application built with [Vite](https://vitejs.dev/). We aim to automate its testing and building in this lab.
```

I don't like `segment` for a wording here, an throughout the other parts, we always called it a `lab`, so I'd stay consistent here 🙂

- [`src/main.tsx`](../src/main.ts) : This is the main entry point of the application.
- [`src/pages/Home.tsx`](../src/pages/Home.tsx) : This route contains most of what you will see upon launching the application.
- [`src/pages/Home.test.ts`](../src/pages/Home.test.tsx) : Here, you will find [`vitest`](https://vitest.dev/) tests that we will run with GitHub Actions.
- [`Dockerfile`](../Dockerfile) : This Docker file packages the application into a container that will be used later in this workshop.

If you want to test the application, you can start a Codespace and run it with the command `npm run dev`
(you can also run the tests with `npm test`). To run the application on your local machine, you will need to install Node.js first.

To test the container, run `docker build . -t local:latest` to build the image and `docker run -p 8080:8080 local:latest` to run it. These commands require a local installation of [Docker](https://www.docker.com/).

## 2 - Implement Continuous Integration (CI)

### 2.1 - Use a starter workflow

> To build a workflow that employs Actions for automating the Continuous Integration process, start by adding a **starter workflow** to your repository:

Reviewer: ```suggestion
To build a workflow that employs Actions for your Continuous Integration process, start by adding a **starter workflow** to your repository:
```

The phrase `automating the CI process` doesn't really make sense to me, as CI is something that cant' be really done manually, can it?

> 1. From your repository's main view, find an navigate to the **Actions** tab.

Reviewer: ```suggestion
1. From your repository's main view, find and navigate to the **Actions** tab.
```

First time I see ChatGPT suggest an actual typo 🙈

Author: It's more likely that I introduced the typo myself. Good catch 🙂

```

The result of your last push to the main branch should resemble the following:

![Actions overview showing a successful workflow run](./images/success-nodejs-workflow.png)

## 3 - Add code coverage to your workflow

> When setting up CI for your project, it's common to provide additional information to users, such as test "*code coverage*".

Reviewer: ```suggestion
When setting up CI for your project, it's common to provide additional information to users, such as the test's "*code coverage*".
```

Author: I believe the original sentence was correct here. "test's code coverage" implies the code coverage is associated with a single test vs. a set of tests.

Author: Possible alternative:

```suggestion
When setting up CI for your project, it's common to provide additional information to users, such as code coverage stats for the project's tests.
```
